### ISSUE TITLE
Release on PYPI

### ISSUE THREAD

**Issue by alejoe91** [CINPLA/spatial-spike-fields#19]
@nicolossus @lepmik since this package has nice functions to compute rate maps, deal with nans, etc I think it should be release to Pypi.

Not that without a release, it cannot be part of an `expipe-plugin-cinpla` release since [this](https://github.com/CINPLA/expipe-plugin-cinpla/blob/main/pyproject.toml#L30) is not supported.

**Comment by nicolossus** [comment]
@alejoe91 Good idea! I also think it might be a good idea to slightly modernize the codebase and generate a documentation page (most functions are already well documented, some streamlining is needed though). A first release could, however, be done with the codebase as it is now.

**Comment by alejoe91** [comment]
I could do a modernization round (pyproject, etc.). Then we just need @lepmik to add the PYPI_API_TOKEN as a secret, so we can auto-release on new tags

**Comment by lepmik** [comment]
Fully support this! Ill add the token
